skyportal/handlers/api/source.py

```diff
@@ -1,18 +1,23 @@
+import datetime
+from functools import reduce
+
 import tornado.web
+from tornado import escape
+
 from sqlalchemy.orm import joinedload
 from sqlalchemy import func, desc
 import arrow
-import datetime
-from functools import reduce
 from marshmallow.exceptions import ValidationError
+
 from baselayer.app.access import permissions, auth_or_token
 from ..base import BaseHandler
 from ...models import (
     DBSession, Comment, Instrument, Photometry, Source, SourceView,
     Thumbnail, GroupSource, Token, User, Group
 )
 from .internal.source_views import register_source_view
-
+from .internal import get_nearby_offset_stars
```

Comment:
See my comments above regarding where these should live

Reply:
Fixed.